src/Ontology/Oep4TxBuilder.cpp

```diff
@@ -0,0 +1,54 @@
+// Copyright © 2017-2020 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "Oep4TxBuilder.h"
+#include "../HexCoding.h"
+
+using namespace TW;
+using namespace TW::Ontology;
```

Comment:
Please avoid using namespace here, include the operation in 

`namespace TW::Ontology {
}`

Reply:
done